src/main/java/org/spongepowered/api/item/inventory/Inventory.java

```diff
@@ -452,27 +475,16 @@
      * @param <T> expected inventory type, specified as generic to allow easy
      *      pseudo-duck-typing
      * @return the query result
-     */
-    <T extends Inventory> T query(String... names);
```

Comment:
I'm confused, why is this going from `String` to `Translation`? The point of deprecating the old methods is to provide an easier upgrade path.